src/main/java/hudson/remoting/Capability.java

```diff
@@ -55,6 +55,8 @@ public boolean supportsPipeThrottling() {
         return (mask& MASK_PIPE_THROTTLING)!=0;
     }
 
+    /** @deprecated no longer used */
+    @Deprecated
```

Comment:
One may still use the Mimic exception if he writes code in the hudson.remoting package. Not sure if it's sane enough BTW. I agree with such comment in Javadoc anyway